sbx/crossq/policies.py

```diff
@@ -0,0 +1,307 @@
+        )(obs, action, train)
+        return q_values
+
+
+class Actor(nn.Module):
+    net_arch: Sequence[int]
+    action_dim: int
+    log_std_min: float = -20
+    log_std_max: float = 2
+    use_batch_norm: bool = True
+    batch_norm_momentum: float = 0.99
+    renorm_warm_up_steps: int = 100_000
+
+    def get_std(self):
+        # Make it work with gSDE
+        return jnp.array(0.0)
+
+    @nn.compact
+    def __call__(self, x: jnp.ndarray, train: bool = False) -> tfd.Distribution:  # type: ignore[name-defined]
+        x = Flatten()(x)
+        if self.use_batch_norm:
+            x = BatchRenorm(
+                use_running_average=not train,
+                momentum=self.batch_norm_momentum,
+                warm_up_steps=self.renorm_warm_up_steps,
+            )(x)
+        else:
+            # Create dummy batchstats
+            BatchRenorm(use_running_average=not train)(x)
+
+        for n_units in self.net_arch:
+            x = nn.Dense(n_units)(x)
+            x = nn.relu(x)
+            if self.use_batch_norm:
+                x = BatchRenorm(use_running_average=not train, momentum=self.batch_norm_momentum)(x)
+
+        mean = nn.Dense(self.action_dim)(x)
+        log_std = nn.Dense(self.action_dim)(x)
+        log_std = jnp.clip(log_std, self.log_std_min, self.log_std_max)
+        dist = TanhTransformedDistribution(
+            tfd.MultivariateNormalDiag(loc=mean, scale_diag=jnp.exp(log_std)),
+        )
+        return dist
+
+
+class CrossQPolicy(BaseJaxPolicy):
+    action_space: spaces.Box  # type: ignore[assignment]
+
+    def __init__(
+        self,
+        observation_space: spaces.Space,
+        action_space: spaces.Box,
+        lr_schedule: Schedule,
+        net_arch: Optional[Union[List[int], Dict[str, List[int]]]] = None,
+        dropout_rate: float = 0.0,
+        layer_norm: bool = False,
+        batch_norm: bool = True,  # for critic
+        batch_norm_actor: bool = True,
+        batch_norm_momentum: float = 0.99,
+        use_sde: bool = False,
+        # Note: most gSDE parameters are not used
+        # this is to keep API consistent with SB3
+        log_std_init: float = -3,
+        use_expln: bool = False,
+        clip_mean: float = 2.0,
+        features_extractor_class=None,
+        features_extractor_kwargs: Optional[Dict[str, Any]] = None,
+        normalize_images: bool = True,
+        optimizer_class: Callable[..., optax.GradientTransformation] = optax.adam,
+        optimizer_kwargs: Optional[Dict[str, Any]] = None,
+        n_critics: int = 2,
+        share_features_extractor: bool = False,
+    ):
+        if optimizer_kwargs is None:
+            # Note: the default value for b1 is 0.9 in Adam.
+            # b1=0.5 is used in the original CrossQ implementation and is found
+            # but shows only little overall improvement.
+            optimizer_kwargs = {}
+            if optimizer_class in [optax.adam, optax.adamw]:
+                optimizer_kwargs["b1"] = 0.5
```

Comment:
Here, the default value of `b1` is set to `0.5` only if no other arguments (or even an empty dict) are passed to the optimizer. It would be cleaner to set the default value to `0.5` regardless of the other optimizer parameters. 

```suggestion
        if optimizer_kwargs is None:
            optimizer_kwargs = {}
        if optimizer_class in [optax.adam, optax.adamw] and "b1" not in optimizer_kwargs:
            # Note: the default value for b1 is 0.9 in Adam.
            # b1=0.5 is used in the original CrossQ implementation but shows only little overall improvement.
            optimizer_kwargs["b1"] = 0.5
```

Reply:
I would keep it as is to be consistent with what is done in the rest of SB3.